templates/openstack/install.html

```diff
@@ -166,21 +166,29 @@ <h2>Large-scale deployment</h2>
 </section>
 
 <section class="p-strip--light">
-  <div class="u-fixed-width">
-    <div class="col-4 u-align--center u-hide--small u-vertically-center">
-
+  <div class="row">
+    <div class="col-5 u-align--center u-hide--small u-vertically-center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/ada1d5b5-operate.svg",
+        alt="",
+        width="228",
+        height="150",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
     </div>
-    <div class="col-6 col-start-large-6 u-vertically-center">
+    <div class="col-7 u-vertically-center">
       <h2>OpenStack tutorials</h2>
       <p>Learn OpenStack through a series of tutorials.</p>
-      <p><a class="p-button--positive js-invoke-modal" href="/openstack/tutorials">Explore tutorials</a></p>
+      <p><a class="p-button--positive" href="/openstack/tutorials">Explore tutorials</a></p>
     </div>
   </div>
 </section>
 
 <section class="p-strip--light">
```

Comment:
```suggestion
<section class="p-strip">
```
Would this be better as it's new section?

Reply:
Good suggestion! I also added a border on the bottom to break the up further.